### PR TITLE
Piecedigital branch

### PR DESCRIPTION
**Pull request by piecedigital**
Transitioned my code from the old file structure to the new.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -33,16 +13,90 @@ var hangmanWords = [
...
+  var result;
+  //checks if the class name is still a button or not
+  if (letter.className === "letter-button") {
+    //if it's still a button it returns the letter
+    //then disables the button
+    result = letter.innerHTML;
+    letter.className = "letter-disabled";
 
+    //calls this function to deal with where the letter goes
+    handlePickedLetter(result);
```
Comment (simon-xiong-ca) [inline scripts.js]: Yes! Exactly how I wanted this to be used :)

File: scripts.js~HEAD [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      letterBlocks[num].appendChild(domElem);
+    });
+  } else {
+    //if letterBlock is not greater than 0 put the letter in the graveyard
+    var domElem = document.createElement("div");
+    domElem.className = "grave-letter";
+    domElem.innerHTML = result;
+    document.getElementById("graveyard").appendChild(domElem);
+  }
+}
```
Comment (simon-xiong-ca) [inline scripts.js~HEAD]: Think this is an old file.

**Comment by simon-xiong-ca** [comment]
This is really good. Are you on Trello? I think others are working on the same parts (displaying the word and verifying letters). Let them know you implemented this so they don't end up doing extra work.

**Comment by piecedigital** [comment]
I move my own card?

**Comment by simon-xiong-ca** [comment]
Yeah I think you can move your own cards. We can talk about moving other cards during our weekly meeting.

**Comment by dsberger** [comment]
hi Daryl, we merged Simon's branch during the hangout and it looks like it created some conflicts for your code. Can you rebase and resolve the conflicts before we merge your branch?

**Pull request merged**